ui/src/app/components/summary/summary.component.html

```diff
@@ -76,15 +76,15 @@
               *ngIf="item.type == 'suggestion'"
               class="suggestion"
             >
-              check_circle
+            wb_incandescent
```

Comment:
The suggestion icon appears to be inverted: https://drive.google.com/file/d/1Uq3ETl1DJ2OPn6lF8gmv2DM0JZzroQLQ/view?usp=sharing&resourcekey=0-fPPXeGS4g8TBjTn8Ig9fdA

Reply:
Updated